packages/flutter/lib/src/material/divider.dart

```diff
@@ -36,37 +36,53 @@ import 'theme.dart';
 /// ![](https://flutter.github.io/assets-for-api-docs/assets/material/divider.png)
 ///
 /// ```dart
-/// Widget build(BuildContext context) {
-///   return Center(
-///     child: Column(
-///       children: <Widget>[
-///         Expanded(
-///           child: Container(
-///             color: Colors.amber,
-///             child: const Center(
-///               child: Text('Above'),
-///             ),
-///           ),
-///         ),
-///         const Divider(
-///           color: Colors.black,
-///           height: 20,
-///           thickness: 5,
-///           indent: 20,
-///           endIndent: 0,
-///         ),
-///         Expanded(
-///           child: Container(
-///             color: Colors.blue,
-///             child: const Center(
-///               child: Text('Below'),
-///             ),
-///           ),
-///         ),
-///       ],
-///     ),
-///   );
-/// }
+///   Widget build(BuildContext context) {
+///    return Center(
+///      child: Column(
+///        children: <Widget>[
+///          Expanded(
+///            child: Container(
+///              color: Colors.amber,
+///              child: const Center(
+///                child: Text('Above'),
+///              ),
+///            ),
+///          ),
+///          const Divider(
+///            color: Colors.black,
```

Comment:
Will this work in dark mode? Maybe use `...colorScheme.onSurface`

Reply:
I removed the color parameter.